apps/omg_eth/test/fixtures.exs

```diff
@@ -64,16 +68,33 @@ defmodule OMG.Eth.Fixtures do
     token_addr
   end
 
+  # inject the exit games into :omg_eth
+  # test fixture does not rely on the release task so it would need this setup
+  defp setup_exit_games() do
```

Comment:
this seems unnecessary.

all contracts are set in test.exs, let's keep it there not to pollute tests and global state (application config).

Reply:
not sure how we do this in `test.exs`. Can the config file load the module we need and make the call to Geth? (Did a bit try after seeing your comment, but it argues those module as unknown)

Alternatively is we add `payment_v2_exit_game` to `localchain_contract_addresses.env` which would need all the change from `plasma-contracts`, `docker-elixir-omg` and I doubt this is what we want, I think we probably prefer to get exit game from `plasma_framework` more?